### DISCUSSION TITLE
Sornar check is not work

### DISCUSSION THREAD

**Discussion by ruanwenjun** [apache/seatunnel#1699] [General]
I find we have the Sornar check in our CI.
@CalvinKirs 

https://github.com/apache/incubator-seatunnel/blob/c3d5f16635246e5a71868f8396612f83a7311fa2/.github/workflows/code-analysys.yml#L36-L40

But it seems it didn't work well on latest PR.
<img width="580" alt="image" src="https://user-images.githubusercontent.com/22415594/163311265-5c8ac955-2b57-402b-838a-64e817189f29.png">

**Comment by CalvinKirs** [comment]
He cannot pass the key, it will only be allowed on the main repository. Operations like dolphin expose the key.

**Reply by ruanwenjun** [reply]
> He cannot pass the key, it will only be allowed on the main repository. Operations like dolphin expose the key.

So, we only has one key and used in two repository?

**Reply by CalvinKirs** [reply]
In simple terms, it will only be triggered every time a Commit (PR merged) is made directly on the main repository, because the secrity cannot be passed, and the PR submitted by the contributor belongs to the fork repository.

**Reply by ruanwenjun** [reply]
Ok, I understand.
